docs/editing_guidelines.md

```diff
@@ -0,0 +1,138 @@
+3. AVOID assertions about structures or cell types that are not part of the cell type being described, except when they pertain to some direct relationships with the cell type being described.
+4. AVOID including details that could better be included in the definition of subtypes of the cell type being described.
+5. LIMIT information that applies to only some members of the class. This should only be used sparingly; when used, it should be made clear that it does not apply to all members of the class.
+6. AVOID using gene expression as the differentium.
+7. AVOID extensive repetition of assertions made in superclass definitions, unless these assertions are used to provide direct evidence for class membership.
+8. DO NOT add informations about what happens in mutant backgrounds or other kind of non-control conditions.
+9. DO NOT include reasons for believing the assertions to be true. These should be recorded in comments.
+10. DO NOT raise questions in the definition. The definition should have the sense of being definitive, which is undermined if we show doubt. Any doubt should be recorded in comments instead.
+
+
+## Comments
+Comments should be used for:
+
+1. Providing evidence. In some cases it is useful to know the type of evidence for an assertion. This should not be recorded in the definition, but _can_ be recorded in a comment.
+2. Disambiguation. Sometimes a single term is used in the literature with multiple meanings. In such cases, acomment should be added outlining these different uses and how they relate to the standard definition set in the ontology.
+3. Reporting editorial decisions (or decisions in waiting) about the term. This includes providing a reason for obsoleting a term, or letting users and curators know that the term may be merged or split in the future, e.g. when enough evidence for the merge/split will be available.
+
+Try to be consistent in how you phrase the various types of comments. For example:
+
+* when giving a reason for obsoletion, use “Obsoleted as ...”;
+* when indicating a potential future merge, use “Possible equivalence with {other term} ...”.
+
+
+## Synonyms
+Extensive addition of synonyms help searching. Add as many synonyms as you like.
+
+Guidelines on the type of synonyms:
+
+1. Use an _exact_ synonym when the name and the synonym can be used interchangeably, at least in a given context. This includes the case where the synonym is an abbreviation that, in a broader context, could have a different meaning. For example, _DPM_  can be an exact synonym of _dorsal pharyngeal muscle cell_, even though “DPM” could have many more meanings generally, because in the context where “dorsal pharyngeal muscle cell” is used, “DPM” is commonly used interchangeably to refer to these cells.
+2. There is no expectation that exact synonyms should be unique across the entire ontology. For example, in the _Drosophila_ Anatomy Ontology _DPM_ is also an exact synonym of _mushroom body dorsal paired medial cell_.
```

Comment:
```suggestion
2. Exact synonyms should be unique across the ontology. In other words, if class _A_ has exact synonym "_X_", "_X_" should not be an exact synonym for any other CL term.
```

Reply:
This requires further discussion.

For what it’s worth, I disagree.

1. Such a policy, if we indeed want to have it, should be enforced in CI checks, not merely stated in guidelines.
2. This would mean that synonyms that may have several meanings in a broader context would all end up as being _related_ synonyms. That category is already the “catch-all” category for all synonyms that are neither exact, narrow, or related, making it the least meaningful category. Such a policy would dilute its meaning even more.
3. This makes even less sense when you consider that ultimately, CL is not a standalone ontology but a part of a much larger set. Enforcing uniqueness of exact synonyms within CL is pointless if we cannot enforce uniqueness OBO-wide.

Reply:
This was discussed and reflected [here](https://github.com/obophenotype/uberon/issues/2363#issuecomment-1446928901). 

>This would mean that synonyms that may have several meanings in a broader context would all end up as being related synonyms.

Yes, annotating terms as exact synonyms should be used conservatively and only when they refer exactly to the concept captured in the ontology.

Reply:
My bad, I must have missed that particular Uberon call.

I still think it’s a bad idea, but if there is already an agreement on the issue then it’s fine by me.

Reply:
But point 1 above still stands: if it is the policy of Uberon/CL that exact synonyms should be unique across the ontology, then the policy should be enforced in CI checks. You cannot rely on editors 1) being aware of the policy (hell, I am an editor and I was not aware of it!) and 2) taking the time to manually check for uniqueness at edit time.

Reply:
@dosumis Well, FBbt and CL already _do_ diverge on that issue. We have a lot of duplicated exact synonyms across the ontology, because we allow synonyms to be “exact in a given context”. And for the record I do not plan to change that in FBbt, whatever CL is doing.

Reply:
For the record, I prefer FBbt's approach but am consistently outvoted in general discussion.  In the end we should judge by some clear statements of use cases.  VFB approach is not great for NLP, but I think is fine for search - users can and should be trusted to do the final disambigutaion.